app/src/main/java/com/hedvig/app/feature/crossselling/ui/CrossSellingResultScreen.kt

```diff
@@ -0,0 +1,180 @@
+package com.hedvig.app.feature.crossselling.ui
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import com.hedvig.app.R
+import com.hedvig.app.ui.compose.HedvigTheme
+import com.hedvig.app.ui.compose.designsystem.LargeContainedButton
+import com.hedvig.app.ui.compose.designsystem.LargeOutlinedButton
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+// TODO text resources still missing
+@Composable
+fun CrossSellingResultScreen(
+    crossSellingResult: CrossSellingResult
+) {
+    HedvigTheme {
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(horizontal = 16.dp)
+        ) {
+            InformationSection(
+                crossSellingResult = crossSellingResult,
+                modifier = Modifier
```

Comment:
Given that the only use-case for `InformationSection` is inside `CrossSellingResultScreen`, perhaps we could inline the modifier? Or is it hoisted because we lack the `BoxScope` inside `InformationSection`?

Reply:
InformationSection could absolutely be an extension function to BoxScope which would allow this. This is in fact how I wrote it initially. The reason the modifier is hoisted is because then by looking at the Box() composable you can immediately know how they are arranged without looking at the two smaller composables themselves. In fact I'd say they themselves shouldn't really care how they are aligned in the box scope, and should just know how to render themselves, and the modifier coming from above can tell them their position in the box scope.
Does the thought process make sense? We can look at it together too and see the alternatives